docs/100_contact_us.md

```diff
@@ -0,0 +1,15 @@
+# Contact Us
+
+If you have a question, check if it is already answered at stackoverflow.com under [cadence-workflow](https://stackoverflow.com/questions/tagged/cadence-workflow) tag.
```

Comment:
check if it is already answered at stackoverflow.com -> check whether it is already answered at stack overflow